test/model/simulator/run.py

```diff
@@ -44,7 +44,7 @@ def build_executable():
 
 def run_simulator(ranks: int, log_level: int, start_port: int, use_udp: bool, kernel_loopback: bool, waveform: bool):
     env = os.environ.copy()
-    env['LD_LIBRARY_PATH'] = f"{os.environ['XILINX_VIVADO']}/lib/lnx64.o"
+    env['LD_LIBRARY_PATH'] += f":{os.environ['XILINX_VIVADO']}/lib/lnx64.o"
```

Comment:
Won't this result in a KeyError if `LD_LIBRARY_PATH` has not been set yet (so doesn't exist in the dictionary)?